src/io.c

```diff
@@ -1399,11 +1399,11 @@ _dispatch_fd_entry_create_with_fd(dispatch_fd_t fd, uintptr_t hash)
 			int result = ioctlsocket((SOCKET)fd, (long)FIONBIO, &value);
 			(void)dispatch_assume_zero(result);
 			_dispatch_stream_init(fd_entry,
-				_dispatch_get_root_queue(DISPATCH_QOS_DEFAULT, false));
+				_dispatch_get_default_queue(false));
```

Comment:
@MadCoder commented on this in the review on the other branch. It's not clear what this change is for.

Reply:
Ah, this is to mirror the changes that you did in the Linux side.

Reply:
https://github.com/apple/swift-corelibs-libdispatch/commit/43a77d2bfae9e729e6de9b5b93ff985a2b5a2ec5#diff-592ee3fd43260e8d9bd24192211bce59L1558